src/utils/hasMethod.js

```diff
@@ -0,0 +1,10 @@
+/**
+ *
+ * @param {object} obj
+ * @param {string} methodName
+ * @return {boolean}
+ */
+function hasMethod(obj, methodName) {
+  return !!obj && typeof obj[methodName] === 'function';
```

Comment:
I would use `Boolean(obj)`

Reply:
Agree, will add it along https://github.com/dashevo/wallet-lib/pull/295#discussion_r701729118